great_tables/_export.py

```diff
@@ -0,0 +1,246 @@
+from __future__ import annotations
+from ._gt_data import GTData
+from ._utils import _try_import
+from typing import Optional
+
+import tempfile
+
+
+def as_raw_html(self: GTData) -> str:
+    """
+    Get the HTML content of a GT object.
+
+    Get the HTML content from a GT object as a string. This function is useful for obtaining the
+    HTML content of a GT object for use in other contexts.
+
+    Parameters
+    ----------
+    gt
+        A GT object.
+
+    Returns
+    -------
+    str
+        An HTML fragment containing a table.
+    """
+    return self._build_data(context="html")._render_as_html()
+
+
+def save(
+    self: GTData,
+    filename: str,
+    path: Optional[str] = None,
```

Comment:
Can we match tools like polars and pandas, and...

* remove the path paramter
* (polars uses the name "file" for filename, and slashes are interpreted as directories; pandas uses the name path_or_buf, which seems a bit gratuitous!)

It seems like the full file path can just be passed straight into PIL. See PIL's [.save() method docs](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.save)